common/src/main/res/values/strings.xml

```diff
@@ -43,4 +43,10 @@
     <!-- Browse Tree -->
     <string name="recommended_title">Recommended</string>
     <string name="albums_title">Albums</string>
+    <string name="recent_title">Recent</string>
+
+    <!-- Recently Played Item Tracking -->
+    <string name="most_recent_item">com.example.android.uamp.recent</string>
+    <string name="most_recent_key">MOST_RECENT_MEDIA_ITEM</string>
+    <string name="most_recent_default">wake_up_01</string>
```

Comment:
Why is it necessary to have a default? If the user hasn't played anything we should probably return an empty list which would stop the media controls from being created (which is the correct behaviour in this case).